tests/openedx_tagging/core/tagging/test_models.py

```diff
@@ -402,3 +466,189 @@ def test_tag_object_invalid_tag(self):
                 "biology101",
             )
         assert "Invalid object tag for taxonomy" in str(exc.exception)
+
+
+@ddt.ddt
+class TestSystemDefinedTaxonomy(TestTagTaxonomyMixin, TestCase):
```

Comment:
If you have time, could you a test for a simple, fully dynamic SystemTaxonomy as @bradenmacdonald [suggested here](https://github.com/openedx/openedx-learning/pull/62#discussion_r1267116903)?